app/services/github_repository_client.rb

```diff
@@ -18,4 +17,16 @@ def repository_views
     response = connection.get { |request| request.params['per'] = 'week' }
     JSON.parse(response.body).with_indifferent_access
   end
+
+  private
+
+  def find_in_locations
+    LOCATIONS.map do |location|
```

Comment:
Instead of a `map`, perhaps we can do `each` and call `break` once we find the file, so that we don't perform unnecessary API calls.

Reply:
to be honest i didn't do the break because i thought it was not a rails way 😆 , i will change it inmediately 👍

Reply:
`detect` might work, too